packages/blockchain/bsc/src/lib/services/bsc.offchain.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  AccountService,
+  ApiServices,
+  Currency,
+  TransferBsc,
+  TransferBscKMS,
+  VirtualCurrencyService,
+  WithdrawalService,
+} from '@tatumio/api-client'
+import {
+  abstractBlockchainOffchain,
+  FromPrivateKeyOrSignatureIdOrMnemonic,
+} from '@tatumio/shared-blockchain-abstract'
+import { evmBasedUtils, EvmBasedWeb3 } from '@tatumio/shared-blockchain-evm-based'
+import { Blockchain } from '@tatumio/shared-core'
+import BigNumber from 'bignumber.js'
+import { bscTxService } from './bsc.tx'
+
+type TransferBscOffchain = FromPrivateKeyOrSignatureIdOrMnemonic<TransferBsc>
+type OffchainResponse = { id?: string; txId?: string; completed?: boolean } | void
+
+const sendBscOffchainTransaction = async (
+  body: TransferBscOffchain,
+  web3: EvmBasedWeb3,
+): Promise<OffchainResponse> => {
+  const txService = bscTxService({ blockchain: Blockchain.BSC, web3 })
+  const { mnemonic, index, privateKey, gasLimit, gasPrice, nonce, ...withdrawal } = body as any
+  const { amount, address } = withdrawal
+  let fromPrivKey: string
+  let txData: any
+
+  if (body.mnemonic && body.index !== undefined) {
+    fromPrivKey = (await ApiServices.blockchain.bsc.bscGenerateAddressPrivateKey({
+      mnemonic: body.mnemonic,
+      index: body.index,
+    })) as string
+  } else {
+    fromPrivKey = body.fromPrivateKey as string
+  }
+
+  const account = await AccountService.getAccountByAccountId(body.senderAccountId)
+  const fee = {
+    gasLimit: gasLimit || '21000',
+    gasPrice: gasPrice || '20',
+  }
+
+  if (account.currency === 'BSC') {
+    txData = txService.native.send.transferSignedTransaction({
+      amount,
+      fromPrivateKey: fromPrivKey,
+      fee,
+      nonce: body.nonce,
+      to: address,
+    })
+  } else {
+    fee.gasLimit = '100000'
+    const vc = await VirtualCurrencyService.getCurrency(account.currency)
```

Comment:
this could be asle supported BEP20 token, not only VC. Take a look at SOL offchain transfer - I cover there USDC_SOL